src/v/pandaproxy/probe.cc

```diff
@@ -13,25 +13,46 @@
 #include "prometheus/prometheus_sanitize.h"
 
 #include <seastar/core/metrics.hh>
+#include <seastar/core/metrics_registration.hh>
 
 namespace pandaproxy {
 
-probe::probe(ss::httpd::path_description& path_desc)
+probe::probe(
+  ss::httpd::path_description& path_desc, const ss::sstring& group_name)
   : _request_hist()
   , _metrics() {
     if (config::shard_local_cfg().disable_metrics()) {
         return;
     }
     namespace sm = ss::metrics;
+    auto operation_label = sm::label("operation");
     std::vector<sm::label_instance> labels{
-      sm::label("operation")(path_desc.operations.nickname)};
+      operation_label(path_desc.operations.nickname)};
+
+    auto aggregate_labels = std::vector<sm::label>{
+      sm::shard_label, operation_label};
+
     _metrics.add_group(
       "pandaproxy",
       {sm::make_histogram(
         "request_latency",
         sm::description("Request latency"),
         std::move(labels),
         [this] { return _request_hist.seastar_histogram_logform(); })});
+
+    _metrics.add_group(
+      group_name,
+      {sm::make_histogram(
+        "request_latency_seconds",
+        sm::description("Request latency"),
+        labels,
+        [this] {
+            // Report from 1ms -> ~1min in seconds
+            return _request_hist.seastar_histogram_logform(
+              16, 1000, 2.0, 1000000);
+        },
```

Comment:
I wonder if it's worth naming this, perhaps in `ssx/metrics.h`

Reply:
Do you mean defining a struct with the default params or something like below?
```
ss::metrics::histogram report_default_histogram(const hdr_hist& hist) {
    static constexpr size_t num_buckets = 16;
    static constexpr size_t first_value = 1000;
    static constexpr double log_base = 2.0;
    static constexpr int64_t scale = 1000000;

    return hist.seastar_histogram_logform(num_buckets, first_value, log_base, scale);
}
```

Reply:
Looks perfect.